worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -78,6 +90,17 @@ public Workflow get(final String workflowName){
 
                 final StringBuilder stringBuilder = new StringBuilder();
                 stringBuilder.append(String.format("var ACTIONS = %s;\n", gson.toJson(workflow.getActions())));
+
+                if (contextScriptFile != null) {
+                    if (!contextScriptFile.exists()) {
```

Comment:
Flip the `if` to avoid the negation:

    if (contextScriptFile.exists()) {
        final String contextScriptFileContent = FileUtils.readFileToString(contextScriptFile, StandardCharsets.UTF_8);
        stringBuilder.append(contextScriptFileContent);
    } else {
        LOG.warn(String.format("Context script file from the path [%s] does not exist.",
                               contextScriptFilePath));
    }